package.json

```diff
@@ -49,7 +49,7 @@
   "license": "Apache-2.0",
   "readmeFilename": "README.md",
   "dependencies": {
-    "@paypal/sdk-constants": "^1.0.94",
+    "@paypal/sdk-constants": "^1.0.97",
```

Comment:
This is fine, but FYI, you don't need to change it since it has the `^` and will automatically update when you reinstall node_modules for minor versions.

Reply:
oh! I forgot that, thanks!